README.md

```diff
@@ -570,6 +570,22 @@ neolink ptz --config=config.toml CameraName zoom 2.5
 
 With 1.0 being normal and 2.5 being 2.5x zoom
 
+### Enabling services (http, https, rtmp, rtsp and onvif)
+
```

Comment:
```suggestion

**For newer wired cameras that have services on the camera hardware. These can be enabled with the services command.**

This is not related to the `rtsp` mode of neolink which is a separate feature. 
```

Reply:
Thanks for the suggestion!
I applied it (and force-pushed), but I took the liberty to remove the "wired" word because in my case (Reolink E1 Zoom) I could enable the http port even though it is also a WiFi camera (that now has a wired port).

Reply:
Oh my wired I meant not battery.

Reply:
Oh, my bad I misunderstood it...
In that case what do you think of:
```suggestion
**For newer cameras (excluding battery-powered cameras) that have services on the camera hardware. These can be enabled with the `services` command.**
```